### ISSUE TITLE
blur in dark mode

### ISSUE THREAD

**Issue by ayushhroyy** [bpwned/protovibrant#3]
Hello, your css is amazing but the issue is in light mode, if you open bookmark bar (command + b) it is blurred and looks beautiful. but if you do the same in dark mode, the effect is not there and it is opaque. this seems to be a minor bug in the dark mode version of this css theme. Thank you have a nice day dattebayo!

**Comment by bpwned** [comment]
You're right, thanks for the report. I will see if I can fix it at some point, but no promises.
